gio/src/auto/dtls_client_connection.rs

```diff
@@ -0,0 +1,205 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::DatagramBased;
+use crate::SocketConnectable;
+use crate::TlsCertificateFlags;
+use glib::object::Cast;
+use glib::object::IsA;
+use glib::signal::connect_raw;
+use glib::signal::SignalHandlerId;
+use glib::translate::*;
+use std::boxed::Box as Box_;
+use std::fmt;
+use std::mem::transmute;
+use std::ptr;
+
+glib::wrapper! {
+    #[doc(alias = "GDtlsClientConnection")]
+    pub struct DtlsClientConnection(Interface<ffi::GDtlsClientConnection, ffi::GDtlsClientConnectionInterface>) @requires DatagramBased;
```

Comment:
... and without the interface, this one (and the server part) is also not really useful.